src/snippets/get-started/sveltekit/Step3.js

```diff
@@ -41,3 +41,30 @@ export async function handle({ event, resolve }) {
 
   return resolve(event);
 }
+
+export async function GET(event) {
+  const decision = await aj.protect(event, { requested: 5 }); // Deduct 5 tokens from the bucket
+  console.log("Arcjet decision", decision);
+
+  if (decision.isDenied()) {
+    if (decision.reason.isRateLimit()) {
+      return error(429, "Too Many Requests");
+    } else if (decision.reason.isBot()) {
+      return error(403, "No Bots Allowed");
+    } else {
+      return error(403, "Forbidden");
+    }
+  }
+
+  return new Response(
+    JSON.stringify({
+      event: event,
+    }),
```

Comment:
`{ message: "Hello World" }`

However, this should use the json helper in sveltekit:
```
import { json } from '@sveltejs/kit';
```
It looks like this is the same import as `error`

Reply:
updated